src/admin-views/list.php

```diff
@@ -1,132 +1,140 @@
-<table id="tribe_ticket_list_table" class="eventtable ticket_list eventForm wp-list-table widefat fixed">
-	<?php
-	global $post;
-	$provider = null;
-	$post_type = 'post';
+<?php if ( ! defined( 'DOING_AJAX' ) || ! DOING_AJAX ) : ?>
+<div id="ticket_list_wrapper">
+<?php endif; ?>
+	<table id="tribe_ticket_list_table" class="eventtable ticket_list eventForm wp-list-table widefat fixed">
+		<?php
+		global $post;
+		$provider = null;
+		$post_type = 'post';
 
-	if ( $post ) {
-		$post_id = get_the_ID();
-		$post_type = $post->post_type;
-	} else {
-		$post_id = $_POST['post_ID'];
+		if ( $post ) {
+			$post_id = get_the_ID();
+			$post_type = $post->post_type;
+		} else {
+			$post_id = $_POST['post_ID'];
 
-		if ( ! empty( $_POST['post_type'] ) ) {
-			$post_type = $_POST['post_type'];
-		} elseif ( ! empty( $_GET['post_type'] ) ) {
-			$post_type = $_GET['post_type'];
+			if ( ! empty( $_POST['post_type'] ) ) {
+				$post_type = $_POST['post_type'];
+			} elseif ( ! empty( $_GET['post_type'] ) ) {
+				$post_type = $_GET['post_type'];
+			}
 		}
-	}
-
-	/**
-	 * Render the ticket row into the ticket table
-	 *
-	 * @param Tribe__Tickets__Ticket_Object $ticket
-	 *
-	 * @since TBD
-	 */
-	function render_ticket_row( $ticket ) {
-		$provider     = $ticket->provider_class;
-		$provider_obj = call_user_func( array( $provider, 'get_instance' ) );
-		?>
-		<tr class="<?php echo esc_attr( $provider ); ?>" data-ticket-order-id="order_<?php echo esc_attr( $ticket->ID ); ?>" data-ticket-type-id="<?php echo esc_attr( $ticket->ID ); ?>">
-			<!-- (handle, name), price, capacity, available, editlink -->
-			<td class=" column-primary ticket_name <?php echo esc_attr( $provider ); ?>">
-				<span class="ticket_cell_label"><?php esc_html_e( 'Ticket Type:', 'event-tickets' ); ?></span>
-				<p><?php echo esc_html( $ticket->name ); ?></p>
-				<button type="button" class="toggle-row"><span class="screen-reader-text"><?php esc_html_e( 'Show more details', 'event-tickets' ); ?></span></button>
-			</td>
 
-			<?php
-			/**
-			 * Allows for the insertion of additional content into the main ticket admin panel after the tickets listing
-			 *
-			 * @param Post ID
-			 * @since TBD
-			 */
-			do_action( 'tribe_events_tickets_ticket_table_add_tbody_column', $ticket, $provider_obj );
+		/**
+		 * Render the ticket row into the ticket table
+		 *
+		 * @param Tribe__Tickets__Ticket_Object $ticket
+		 *
+		 * @since TBD
+		 */
+		function render_ticket_row( $ticket ) {
```

Comment:
Tend to think we shouldn't be placing functions inside of views (even admin views). Can this be relocated, perhaps as a method on an object that's within the view's scope?